### DISCUSSION TITLE
How to link to image inside of ADR?

### DISCUSSION THREAD

**Discussion by qrush** [thomvaill/log4brains#43] [Q&A]
Is there a way we could easily copy over a jpg, png, etc with an ADR? It seems like the build/preview process ignores images.

**Comment by MichaelBaySAP** [comment]
+1

**Comment by borisnikolicdoktorse** [comment]
Can we have an answer please? @thomvaill 

Thanks!

**Comment by previousdeveloper** [comment]
+1

**Comment by ck3mp3r** [comment]
Definitely not just a "nice to have"...
